com.unity.shadergraph/Editor/GraphUI/DataModel/Examples/ShaderGraphExampleTypes.cs

```diff
@@ -218,7 +202,7 @@ public object ObjectValue
                     default:
                         switch (GetPrimitive())
                         {
-                            case GraphType.Primitive.Int: sc(0, (float)value); return;
+                            case GraphType.Primitive.Int: sc(0, Convert.ToSingle(value)); return;
```

Comment:
Fun type headache: if we get an `int` as an `object`, we can't directly cast it to `float`.